policy/examples/requirements.md

```diff
@@ -579,3 +580,74 @@ Version 1.1.0 for 2 providers requiring Provider `/status_changes` with the mini
 ```
 
 [Top](#table-of-contents)
+
+## GBFS Only
+
+Since Requirements allows the GBFS versions and optional endpoints and fields to be defined, an agency could use it to only require public GBFS feeds, and not require MDS at all.
+
+```json
+{
+  "metadata": {
+    "mds_release": "1.2.0",
+    "version": "2",
+    "last_updated": "1611958740",
+    "max_update_interval": "T1M",
+    "agency_uuid": "737a9c62-c0cb-4c93-be43-271d21b784b5",
+    "agency_name": "Louisville Metro",
+    "agency_timezone": "America/New_York",
+    "agency_language": "en-US",
+    "agency_currency": "USD",
+    "agency_website_url": "https://www.cityname.gov/transporation/",
+    "url": "https://mds.cityname.gov/policy/requirements/1.2.0"
```

Comment:
url path should be 2?

Reply:
What do you mean by "2"?

Reply:
ie https://mds.cityname.gov/policy/requirements/2 – I was imagining the URL path would include the version of the file rather than the version of the requirements spec

Reply:
I think the path should be static and not change with each edit, though it could change with each MDS version. That's why there is a 'file_version' field and 'last_updated' field, and why OMF will host files in GitHub for a built in version history. Even if it did change with each edit, I think it would be `https://mds.cityname.gov/policy/requirements/1.2.0/2`. Even so, I'd rather see the latest version stay at the same URL, and maybe older version can have their URLs updated in a consistent way, eg v1 would show up at `https://mds.cityname.gov/policy/requirements/1.2.0/1` when v2 is made. But really, this is just an example, and agencies can do what they wish.

Maybe we need some additional language about this, and also recommend agencies host on a system that tracks versions publicly?

Reply:
Got it. Sounds like I was making the wrong assumption, which is that Agencies would host the old versions of their requirements files.